### PR TITLE
Update to svelte 4

### LINKED ISSUE #17: Unmet peer dependency - Svelte version 4.0.0
Warning: svelte-material-icons package requires a peer dependency of svelte version ^3.0.0, when using version 4.0.0 of svelte.

Updating the svelte peerDependencies field in your package.json file should solve this issue. If there isnt other breaking changes you need to take account for.

### PR DESCRIPTION
**Pull request by ranile**
Fixes #17

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,6 +22,6 @@
     "cheerio": "^1.0.0-rc.3"
   },
   "peerDependencies": {
-    "svelte": "^3.0.0"
+    "svelte": "^4.0.0"
```
Comment (tifosiblack) [inline package.json:25]: maybe 3.x || 4.x  to support both ?

Comment (alextran1502) [reply package.json:25]: You can do this 

```json
  "peerDependencies": {
    "svelte": "^3.0.0 || ^4.0.0"
  }
```

**Comment by alextran1502** [comment]
@ramiroaisen Hello, thank you for the library! Would it be possible for you to help merge and publish a new release to support Svelte 4? Thank you

**Comment by ramiroaisen** [comment]
Hello, thanks for your contributions, I added 3.0.0 || 4.0.0 as asked to support both versions, I'm merging this now.
Thank you very much!

Published on npm as version 3.0.5

**Pull request merged**